microdata_tools/validation/model/metadata.py

```diff
@@ -178,16 +191,20 @@ def raise_invalid_with_unit_type(field_name: str):
                 "together with a unitType"
             )
 
-        if values.get("unitType", None) is not None:
-            if values.get("dataType", None) is not None:
+        valuedomain = values.get("valueDomain", None)
```

Comment:
I like this refactor :100:
but could you snake case the names like: value_domain, data_type and unit_type 🙇🏻 ?